### ISSUE TITLE
No DINERS check

### ISSUE THREAD

**Issue by Dekari** [Mangopay/cardregistration-js-kit#42]
https://github.com/Mangopay/cardregistration-js-kit/blob/c7c96336ee37893468a8984523439a60bf1e4974/kit/mangopay-kit.js#L301

Here should check for DINERS card also.

if(cardType === "MAESTRO" || cardType === "BCMC") { 

if(cardType === "MAESTRO" || cardType === "BCMC" || cardType === "DINERS") {

**Comment by mangomaxoasis** [comment]
Hi @Dekari ,
For security reasons, we request the CVV of the DINERS card.
Regards,
Max

**Comment by frossigneux** [comment]
Please fix this code line https://github.com/Mangopay/cardregistration-js-kit/blob/master/kit/mangopay-kit.js#L278
The issue has been closed but not fixed...

**Comment by mangomaxoasis** [comment]
Hi @frossigneux 

The modification you are proposing removes the CVV check for Diners (as it is removed for MAESTRO and BCMC)
 
               _if(cardType === "MAESTRO" || cardType === "BCMC") {
                   return true;
               }
               cvv = cvv ? cvv.trim() : "";_

Diner's has the possibility of CVV checks, but these are implemented in a non-obligatory and sometimes error prone way -> it is an ongoing we are having with them. However, we have included CVV check in out SDK by default for security reasons. This is why the ticket was closed. 

Diner's can function without this check, if you would like to remove it. We will keep our community up to date with evolutions on both CVV and 3DS with relation to Diners. 

Regards,
Max
